dymos/examples/brachistochrone/test/test_brachistochrone_path_constraint.py

```diff
@@ -0,0 +1,75 @@
+import os
+import unittest
+
+import matplotlib
+import openmdao.api as om
+import matplotlib.pyplot as plt
+import dymos as dm
+
+from openmdao.utils.testing_utils import use_tempdirs
+from openmdao.utils.assert_utils import assert_near_equal
+from dymos.examples.brachistochrone.brachistochrone_ode import BrachistochroneODE
+
+matplotlib.use('Agg')
+plt.style.use('ggplot')
+
+
+@use_tempdirs
+class TestBrachistochroneExprPathConstraint(unittest.TestCase):
+
+    def _make_problem(self, tx):
+        p = om.Problem(model=om.Group())
+
+        p.driver = om.ScipyOptimizeDriver()
+        p.driver.declare_coloring()
+
+        phase = dm.Phase(ode_class=BrachistochroneODE,
+                         transcription=tx)
+
+        p.model.add_subsystem('phase0', phase)
+
+        phase.set_time_options(fix_initial=True, duration_bounds=(.5, 10))
+
+        phase.set_state_options('x', fix_initial=True)
+        phase.set_state_options('y', fix_initial=True)
+        phase.set_state_options('v', fix_initial=True)
+
+        phase.add_control('theta', continuity=True, rate_continuity=True, opt=True,
+                          units='deg', lower=0.01, upper=179.9, ref=1, ref0=0)
+
+        phase.add_boundary_constraint('x', loc='final', equals=10.0)
+        phase.add_path_constraint('pc = y-x/2-1', lower=0.0)
+
```

Comment:
Is the Left Hand Side always required? maybe that info should be presented in the docstring for "name".